build/bin/user_setup

```diff
@@ -1,13 +0,0 @@
-#!/bin/sh
-set -x
-
-# ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
-mkdir -p ${HOME}
-chown ${USER_UID:-1001}:0 ${HOME}
-chmod ug+rwx ${HOME}
-
-# runtime user will need to be able to self-insert in /etc/passwd
-chmod g+rw /etc/passwd
```

Comment:
I'm curious if this part is still possibly. In theory - `/etc/passwd` should be 0600 with the new approach, right?